census/core.py

```diff
@@ -116,14 +116,46 @@ def tables(self, year=None):
 
         # Pass it out
         return resp.json()['groups']
+    
+    def variables(self, year=None) -> list: # added
```

Comment:
Great idea here. We do plan on taking on a more general project of fetching API metadata from the 
[Census Data API Discovery Tool](https://www.census.gov/data/developers/updates/new-discovery-tool.html). Would you mind separating this out into a separate PR?